client/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/style.css">
+    <link rel="icon" href="./assets/images/favicon.ico">
+    <script defer src="./assets/script.js"></script>
+    <script defer src="./assets/script.js"></script>
```

Comment:
Here you have called for the script.js file twice. This generates an error because I think it loads it twice and then you are trying to reassign a const

Reply:
oops didn't realise, that was an accidental duplicate. thanks again